src/webui/api/torrentscontroller.cpp

```diff
@@ -567,6 +567,7 @@ void TorrentsController::addAction()
     const TriStateBool rootFolder = parseTriStateBool(params()["root_folder"]);
     const QString savepath = params()["savepath"].trimmed();
     const QString category = params()["category"];
+    const QSet<QString> tags = QSet<QString>::fromList(params()["tags"].split(',', QString::SkipEmptyParts));
```

Comment:
`QSet<>::fromList` is deprecated.
You should use: `const QSet<QString> tags = List::toSet(params()["tags"].split(',', QString::SkipEmptyParts));`